src/core/a-scene.js

```diff
@@ -93,24 +89,12 @@ var AScene = module.exports = registerElement('a-scene', {
         var resizeCanvas = this.resizeCanvas.bind(this);
         this.setupKeyboardShortcuts();
         this.attachFullscreenListeners();
```

Comment:
Not sure, debatable I suppose.  my thought was that if you are going to manage your own viewport rendering, then you would want to also manually handle fullscreen events as well.